rust/connlib/shared/src/device_id.rs

```diff
@@ -3,8 +3,6 @@ use std::fs;
 use std::io::Write;
 
 pub struct DeviceId {
-    /// True iff the device ID was not found on disk and we had to generate it, meaning this is the app's first run since installing.
-    pub is_first_time: bool,
```

Comment:
Without sudo powers, the GUI can't write this file, so I use a different file to do the first-launch behavior.